routes/spec.graphql

```diff
@@ -28,7 +28,7 @@ type Query {
   """
   guild(
     """
-    Username or non-dashed UUID of player. UUID should be used when feasible for better performance.
+    Username or non-dashed UUID of player. UUID should be used when feasible for better performance. Will use the Guild's Name if you use the syntax name/{guildName} to it.
```

Comment:
You should make a new method, named something like `guild_by_name`, then copy and paste the code from above and change `player_name` to `guild_name` + add an entry here

Reply:
Also sorry for not making a more thorough review the first time :P

Reply:
Its fine this is my first time working with an API so i didnt know exactly how to make it lol.